src/genie/libs/parser/iosxe/tests/ShowApDot115GhzChannel/cli/equal/golden_output_expected.py

```diff
@@ -0,0 +1,28 @@
+expected_output = {
+    "channel_assignment": {
+        "chan_assn_mode": "AUTO",
+        "chan_upd_int": 12,
+        "anchor_time_hour": 7,
+        "channel_noise": "Enable",
+        "channel_interference": "Enable",
+        "channel_load": "Disable",
+        "device_aware": "Disable",
+        "clean_air": "Disabled",
+        "wlc_leader_name": "sj-00a-ewlc1",
+        "wlc_leader_ip": "10.7.5.133",
+        "last_run_seconds": 15995,
+        "dca_level": "MEDIUM",
+        "dca_db": 15,
+        "chan_width": "80 MHz",
```

Comment:
let's have only number as same as line 17.
```
"chan_width_mhz": 80,

Reply:
please ignore. same added to https://github.com/CiscoTestAutomation/genieparser/pull/251